### ISSUE TITLE
FAIL: //tensorflow/python/debug:analyzer_cli_test

### ISSUE THREAD

**Issue by lukeiwanski** [lukeiwanski/tensorflow#40]
## System Info

```
  Name:						 Hawaii
  Vendor:					 Advanced Micro Devices, Inc.
  Device OpenCL C version:			 OpenCL C 2.0 
  Driver version:				 1912.5 (VM)
  Profile:					 FULL_PROFILE
  Version:					 OpenCL 2.0 AMD-APP (1912.5)
  Extensions:					 cl_khr_fp64 cl_amd_fp64 cl_khr_global_int32_base_atomics cl_khr_global_int32_extended_atomics cl_khr_local_int32_base_atomics cl_khr_local_int32_extended_atomics cl_khr_int64_base_atomics cl_khr_int64_extended_atomics cl_khr_3d_image_writes cl_khr_byte_addressable_store cl_khr_gl_sharing cl_khr_gl_depth_images cl_ext_atomic_counters_32 cl_amd_device_attribute_query cl_amd_vec3 cl_amd_printf cl_amd_media_ops cl_amd_media_ops2 cl_amd_popcnt cl_khr_image2d_from_buffer cl_khr_spir cl_khr_subgroups cl_khr_gl_event cl_khr_depth_images cl_khr_mipmap_image cl_khr_mipmap_image_writes

```
ComputeCpp 0.2.0

## To reproduce
```
bazel test --config=sycl --local_test_jobs=4 -k --test_lang_filters=cc,py --action_env=LD_PRELOAD=/usr/lib/libOpenCL.so.1 --test_timeout 300,750,1200,3600 //tensorflow/python/debug:analyzer_cli_test
```

## Error
```
FAIL: testPrintTensor (__main__.AnalyzerCLISimpleMulAddTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/luke/.cache/bazel/_bazel_luke/6b8509cbd1cb09897b517a53b72ea21d/execroot/tensorflow/bazel-out/local_linux-opt/bin/tensorflow/python/debug/analyzer_cli_test.runfiles/org_tensorflow/tensorflow/python/debug/cli/analyzer_cli_test.py", line 933, in testPrintTensor
    ], out.lines)
AssertionError: Lists differ: ['Tensor "simple_mul_add/matmu... != ['Tensor "simple_mul_add/matmu...

First differing element 4:
array([[ 7.],
array([[  1.00000000e+001],

  ['Tensor "simple_mul_add/matmul:0:DebugIdentity":',
   '  dtype: float64',
   '  shape: (2, 1)',
   '',
-  'array([[ 7.],',
-  '       [-2.]])']
+  'array([[  1.00000000e+001],',
+  '       [  6.93168824e-310]])']

```

**Comment by jeffmak** [comment]
I am currently getting a different error for this test:
`
ERROR: setUpClass (__main__.AnalyzerCLIWhileLoopTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jeffrey/.cache/bazel/_bazel_jeffrey/f9d61352387e90491903f72fd157321c/execroot/tensorflow/bazel-out/local_linux-opt/bin/tensorflow/python/debug/analyzer_cli_test.runfiles/org_tensorflow/tensorflow/python/debug/cli/analyzer_cli_test.py", line 1862, in setUpClass
    cls._dump_root, partition_graphs=run_metadata.partition_graphs)
  File "/home/jeffrey/.cache/bazel/_bazel_jeffrey/f9d61352387e90491903f72fd157321c/execroot/tensorflow/bazel-out/local_linux-opt/bin/tensorflow/python/debug/analyzer_cli_test.runfiles/org_tensorflow/tensorflow/python/debug/lib/debug_data.py", line 690, in __init__
    self._load_all_device_dumps(partition_graphs, validate)
  File "/home/jeffrey/.cache/bazel/_bazel_jeffrey/f9d61352387e90491903f72fd157321c/execroot/tensorflow/bazel-out/local_linux-opt/bin/tensorflow/python/debug/analyzer_cli_test.runfiles/org_tensorflow/tensorflow/python/debug/lib/debug_data.py", line 712, in _load_all_device_dumps
    self._load_partition_graphs(partition_graphs, validate)
  File "/home/jeffrey/.cache/bazel/_bazel_jeffrey/f9d61352387e90491903f72fd157321c/execroot/tensorflow/bazel-out/local_linux-opt/bin/tensorflow/python/debug/analyzer_cli_test.runfiles/org_tensorflow/tensorflow/python/debug/lib/debug_data.py", line 1009, in _load_partition_graphs
    self._validate_dump_with_graphs(device_name)
  File "/home/jeffrey/.cache/bazel/_bazel_jeffrey/f9d61352387e90491903f72fd157321c/execroot/tensorflow/bazel-out/local_linux-opt/bin/tensorflow/python/debug/analyzer_cli_test.runfiles/org_tensorflow/tensorflow/python/debug/lib/debug_data.py", line 1245, in _validate_dump_with_graphs
    if self.node_op_type(recipient) == "Merge":
  File "/home/jeffrey/.cache/bazel/_bazel_jeffrey/f9d61352387e90491903f72fd157321c/execroot/tensorflow/bazel-out/local_linux-opt/bin/tensorflow/python/debug/analyzer_cli_test.runfiles/org_tensorflow/tensorflow/python/debug/lib/debug_data.py", line 1692, in node_op_type
    return self._node_op_types[device_name][node_name]
KeyError: u'/job:localhost/replica:0/task:0/device:SYCL:0'
`

**Comment by jwlawson** [comment]
Yeah I've fixed the KeyError in #123.

**Comment by jwlawson** [comment]
Fixed by #151
